umodbus/serial.py

```diff
@@ -263,8 +395,8 @@ def _send(self, modbus_pdu: bytes, slave_addr: int) -> None:
             self._ctrlPin(0)
 
     def _send_receive(self,
-                      modbus_pdu: bytes,
                       slave_addr: int,
+                      modbus_pdu: bytes,
```

Comment:
good finding, this is one of the reasons to use named parameters :)

Reply:
Haha, you're right!